Automattic/sensei
# Add importer done page

### Changes proposed in this Pull Request

* Add components to render the Done page and Import log
* Add done step state to store with results and import log
* Add restart importer action that resets state
* Adjust progress polling to dispatch an action instead of step data invalidation and resolver - so it also updates the done step
* CSS: Make setup wizard grey links explicit so they are not the default

### Testing instructions

* Import some content (questions) without errors
* Done page should show success, and the right number of imported questions
* `Import More Content` should go back to the first step and allow a new import
* Import a questions csv with errors (for example, invalid media attachments)
* Done page should show a warning, the right number for successful and failed imports
* Import log should show detailed description for the affected entries


### Screenshot / Video

![image](https://user-images.githubusercontent.com/176949/86045471-f68d7280-ba4b-11ea-9bba-9c4d3039877b.png)

![image](https://user-images.githubusercontent.com/176949/86046424-7f58de00-ba4d-11ea-9a85-ffd961c1eacb.png)
